woo-razorpay.php

```diff
@@ -1079,7 +1079,7 @@ private function enqueueCheckoutScripts($data)
                     array('razorpay_checkout'));
 
                 wp_register_script('razorpay_checkout',
-                    'https://checkout.razorpay.com/v1/checkout.js',
+                    'https://checkout.razorpay.com/v1/checkout-1cc.js',
```

Comment:
@angbpy This is not 1cc related. It's for native. You need to change the const value for this `RZP_CHECKOUTJS_URL`

Reply:
Alright making the changes